app/javascript/packages/phone-input/index.ts

```diff
@@ -22,14 +24,6 @@ interface IntlTelInput extends IntlTelInputPlugin {
   options: Options;
 }
 
-const isPhoneValid = (phone, countryCode) => {
```

Comment:
It's not clear to me why we're removing this. It seemed to add some additional tolerances for assuming a U.S. number, which I think we'd want.

Reply:
Although maybe it's not strictly necessary 🤔 I was testing this a bit and it seems like `isValidNumber` is pretty forgiving in the types of numbers it receives, so trying with the `+1` prefix shouldn't be necessary.

```
> require('libphonenumber-js').isValidNumber('5135550100', 'US')
true
> require('libphonenumber-js').isValidNumber('15135550100', 'US')
true
> require('libphonenumber-js').isValidNumber('+15135550100', 'US')
true
> require('libphonenumber-js').isValidNumber('+1 5135550100', 'US')
true
```

Reply:
Yeah, I found that it wasn't a strict requirement, so I got rid of it.